Cargo.toml

```diff
@@ -35,6 +35,9 @@ global-context = ["std"]
 # and is not necessary.)
 global-context-less-secure = ["global-context"]
 
+# Enables debug logging, do not enable in production systems.
+danger_leak_secret_material = ["std"]
```

Comment:
In a84629c3cff592cf1799b50fb60ea8e052c565fa and 8cb37e093bf91bd1f621af5f35ca473153bd5884:

As discussed elsewhere we cannot use a cargo feature for this.

Reply:
My bad, I had not worked on this since all our chats. Sorry for not making that explicit and wasting your review time.